ports/stduuid/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "stduuid",
+  "version-string": "2019-02-23",
+  "description": "A C++17 cross-platform single-header library implementation for universally unique identifiers, simply know as either UUID or GUID (mostly on Windows)",
+  "homepage": "https://github.com/mariusbancila/stduuid",
+  "license": "MIT",
+  "features": {
```

```suggestion
  "dependencies":[
  {
    "name": "vcpkg-cmake",
    "host": true
  },
   {
    "name": "vcpkg-cmake-config",
    "host": true
  },  
  ],
  "features": {
```
We should add the dependencies if using new functions.